AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/holder/EViewHolder.java

```diff
@@ -159,4 +162,28 @@ public JFieldVar getAlreadyInflated() {
 	private void setAlreadyInflated() {
 		alreadyInflated = generatedClass.field(PRIVATE, getCodeModel().BOOLEAN, "alreadyInflated" + generationSuffix(), JExpr.FALSE);
 	}
+
+	@Override
+	public JBlock getOnViewChangedBodyAfterInjectionBlock() {
+		JBlock onViewChangedBodyAfterInjectionBlock = super.getOnViewChangedBodyAfterInjectionBlock();
+
+		for (OnFinishInflateCallBlock callBlock : onFinishInflateCallBlock) {
```

Comment:
This should only happen at the first call to `getOnViewChangedBodyAfterInjectionBlock()`.